daemon/create.go

```diff
@@ -70,6 +76,67 @@ func (daemon *Daemon) ContainerCreate(job *engine.Job) error {
 	return nil
 }
 
+func validateVolumePath(config *runconfig.Config, hostConfig *runconfig.HostConfig) error {
+	validModes := map[string]bool{
+		"rw": true,
+		"ro": true,
+	}
+
+	//validate bind volums
+	for _, spec := range hostConfig.Binds {
+		var (
+			path, mountToPath, mode string
+			arr                     = strings.Split(spec, ":")
+		)
+		switch len(arr) {
```

Comment:
I still think this should be using parseBindMountSpec, and keep the error there, just check for error here.